### DISCUSSION TITLE
how to debug livewire error using web exception page?

### DISCUSSION THREAD

**Discussion by shakib04** [livewire/livewire#8856] [1. Help]
When something wrong in my livewire component and blade file i get this error. Problem is It is not helpful to debug the problem. I need to look into laravel.log file for finding the issue.

![image](https://github.com/user-attachments/assets/6a157c5b-8f81-40b8-904b-7c24d30d1ff5)

**Comment by PhiloNL** [comment]
Duplicate of https://github.com/livewire/livewire/discussions/8637
